File: app/templates/server_node/koa/server_node_koa_babel/api/todo/route/todo-route.js
"use strict";

import TodoController from '../controller/todo-controller';

export default class TodoRoutes {
    static init(router) {
      let _todoController = new TodoController();

      router.get('/api/todos', _todoController.getAll);
      router.post('/api/todos', _todoController.createTodo);
      router.del('/api/todos/:id', _todoController.deleteTodo);
    }
}

File: app/templates/server/go/gin/api/todo/route/todoroute.go
package todoroute

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/todo/controller"
	"gopkg.in/gin-gonic/gin.v1"
)

func Init(r *echo.Echo) {
	r.GET("/api/todos", todocontroller.GetAll)
	r.GET("/api/todos/:id", todocontroller.GetById)
	r.POST("/api/todos", todocontroller.NewTodo)
	r.DELETE("/api/todos/:id", todocontroller.RemoveTodo)
}
